blog-service/2023-05-19-apps.md

```diff
@@ -0,0 +1,16 @@
+---
+title: May 19, 2023 (Apps)
+image: https://www.sumologic.com/img/logo.svg
+keywords:
+  - apps
+  - app catalog
+  - saas cloud
+hide_table_of_contents: true
+authors:
+  - url: https://help.sumologic.com/release-notes-service/rss.xml
+    image_url: /img/release-notes/rss-orange.png
+---
+
+We're excited to announce the release of the new Microsoft Graph Security app for Sumo Logic. This app leverages Sumo Logic's [Cloud-to-Cloud Microsoft Graph Security source](/docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/microsoft-graph-security-api-source) identify security threats and helps you to improve web security posture, proactively detect and mitigate potential security risks, and improve compliance with security policies and regulations. Here are some of the key features that this app offers:
```

Comment:
removed this since it's blank underneath

```suggestion
We're excited to announce the release of the new Microsoft Graph Security app for Sumo Logic. This app leverages Sumo Logic's [Cloud-to-Cloud Microsoft Graph Security source](/docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/microsoft-graph-security-api-source) identify security threats and helps you to improve web security posture, proactively detect and mitigate potential security risks, and improve compliance with security policies and regulations.
```